doc/error_management.md

```diff
@@ -115,9 +115,9 @@ println!(
 );
 ```
 
-### getting more information: nom::error::VerboseError
+### getting more information: nom-language::error::VerboseError
 
-The  `VerboseError<I>` type accumulates more information about the chain of
+The  `VerboseError<I>` type (available from the companion `nom-language` crate) accumulates more information about the chain of
```

Comment:
Thanks for making this PR, I've come here after trying to use the VerboseError from nom :)

Just a minor suggestion: that the struct comes from a different crate is an important piece of information, I'd not put it in a parenthetical.  Maybe something like this instead:

The  companion `nom-language` crate's `VerboseError<I>` type accumulates